perfluoroethers/perfluoroethers.py

```diff
@@ -0,0 +1,23 @@
+import os
+import glob
+from pkg_resources import resource_filename
+
+from foyer import Forcefield
+
+
+def get_ff_path():
+    return [resource_filename('perfluoroethers', 'xml')]
+
+
+def get_perfluoroether_forcefield_path():
+    for dir_path in get_ff_path():
+        file_pattern = os.path.join(dir_path, '*.xml')
+        file_paths = [file_path for file_path in glob.glob(file_pattern)]
+    return file_paths
+
+
+def get_perfluoroether_forcefield():
+    return Forcefield(get_perfluoroether_forcefield_path())
+
+
+PFE = get_perfluoroether_forcefield()
```

Comment:
I share the same concerns as @ahy3nz in https://github.com/mosdef-hub/foyer/pull/282#issuecomment-547144491.
Maybe we could do something like this:
```suggestion
PFE = get_perfluoroether_forcefield
```
Then, we can call the function, `foyer.Forcefields.PFE()` to get an instance of the forcefield, still in a concise way. I'm guessing this will also clean up the [warnings](https://github.com/mosdef-hub/foyer/pull/282#issuecomment-546546417) that are outputted upon doing `import foyer`

Reply:
You would have to change this to the OPLS and TRAPPE forcefields in foyer too:
https://github.com/mosdef-hub/foyer/blob/71741fda7b7230fabc535d47569ae27746125314/foyer/forcefields/forcefields.py#L30-L31